MAINTAINERS.yml

```diff
@@ -32,6 +32,9 @@
 #     labels:
 #         List of GitHub labels to add to pull requests that modify the area.
 #
+#     tests:
+#         List of test identifiers to be run when this area is affected by a PR.
```

Comment:
Can you elaborate what do you mean with test identifiers?
Will all twister tests which identifier start with this prefix(es) be run?